src/default/assets/css/elements/_navigation.sass

```diff
@@ -115,6 +115,11 @@
     position: -webkit-sticky
     position: sticky
     top: .5rem
+    transition: .3s
+
+    &.tsd-navigation--displace
+        max-height: calc(100vh - 1rem - #{$TOOLBAR_HEIGHT})
+        top: calc(.5rem + #{$TOOLBAR_HEIGHT});
```

Comment:
Semicolon shouldn't be necessary, this is sass, not scss.

Reply:
Right, thanks